### PR TITLE
New storage configuration options

### PR DESCRIPTION
**Pull request by tisnik**
# Description

New storage configuration options

## Type of change

- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Refactor (refactoring code, removing useless files)
- Unit tests (no changes in the code)
- Documentation update

## Testing steps

To be checked on CI.

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

**Review by matysek — APPROVED** [review]
/lgtm

File: config.toml [hunk 1]
```diff
@@ -33,6 +33,15 @@ sqlite_datasource = "./aggregator.db"
 log_sql_queries = true
 type = "sql"
 
+[dvo_recommendations_storage]
+db_driver = "sqlite3"
+sqlite_datasource = "./dvo.db"
```
Comment (Bee-lee) [inline config.toml:38]: IMHO I don't think it makes much sense to support sqlite. It isn't supported at all for the "old" storage. In fact you can't use it as migrations will fail

**Review by Bee-lee — APPROVED** [review]
LGTM apart of the sqlite support

File: config.toml (hunk 1 above)
Comment (tisnik) [reply config.toml]: removed

**Review by juandspy — APPROVED** [review]

**Pull request merged**